### ISSUE TITLE
ApiListener: Metrics for cluster data

### ISSUE THREAD

**Issue by lazyfrosch** [Icinga/icinga2#5133]
It would be nice to have an indicator inside `ApiListener` or `JsonRpcConnection` about the amount of events send and received.

Probably split by event type (API function).

Would also give a nice indicator for #5132

**Comment by dnsmichi** [comment]
Some metrics we already collect: 

* json_rpc
  * clients
  * work_queue_items - sum up all work queues?
  * sync_queue_items
  * relay_queue_items
* http
  * clients

Changes depend on #5265 (this shouldn't be backported anyways). It also fixes the wrong performance data for the ApiListener in /v1/status and further allows us to forward PerfdataValue values to our metric writers.

The `cluster` check includes the performance data from ApiListener::StatsFunc(). 

I'm not satisfied with the way the json rpc workqueues are fetched and how their stats are calculated. I.e. sum up all work queue items for just one metric? The number of work queues depends on the underlaying cpu cores.

**Comment by dnsmichi** [comment]
`work_queue_items` sums up all work queues (this is dependent on the number of availably cpus and differs on each system). `work_queue_count` gives and indicator how many work queues are in place.

The `cluster` check now includes all the metrics from ApiListener::StatsFunc(). The `icinga` check does not provide any feature stats, that will be an addition.

One thing we should also add - work queue rates. That's another PR I'm going to submit soon.
